signals/parser/schema.py

```diff
@@ -28,6 +29,25 @@ def create_objects(self, object_json):
             data_object = DataObject(object_name, data_object_json[object_name])
             self.data_objects[data_object.name] = data_object
 
+    # Note: Why do we call them "apis" in method names, but then call them "urls" in the schema/code
```

Comment:
Also, I'm assuming you did this out here because you didn't have access to the `data_objects` inside of the API objects?